utils/scripts/gen_tflite_models.py

```diff
@@ -373,6 +373,67 @@ def gen_binary_operator_test(name, type, input_shape):
 gen_binary_operator_test(name="min", type="min", input_shape=(1, 10))
 
 
+# ----------------------------------------------------------------------------------------------------------------------
+#                                            Binary broadcasted operators
+# ----------------------------------------------------------------------------------------------------------------------
+def gen_binary_broadcast_operator_test(name, type, shape_1, shape_2):
+    # Create model.
+    inp1 = layers.Input(name="input1", batch_size=shape_1[0], shape=shape_1[1:])
+    inp2 = layers.Input(name="input2", batch_size=shape_2[0], shape=shape_2[1:])
+    if type == "add":
+        out = tf.math.add(inp1, inp2)
+    elif type == "mul":
+        out = tf.math.multiply(inp1, inp2)
+    elif type == "sub":
+        out = tf.math.subtract(inp1, inp2)
+    elif type == "div":
+        out = tf.math.divide(inp1, inp2)
+    elif type == "max":
+        out = tf.math.maximum(inp1, inp2)
+    elif type == "min":
+        out = tf.math.minimum(inp1, inp2)
+    else:
+        print('Binary operator "%s" not supported!' % type)
+        exit(1)
+    model = Model(inputs=[inp1, inp2], outputs=[out])
+    # Create data.
+    np.random.seed(0)
+    inp1_tensor = np.random.rand(*shape_1).astype(np.float32)
+    inp2_tensor = np.random.rand(*shape_2).astype(np.float32)
+    if type == "pow":
+        inp1_tensor = np.abs(inp1_tensor) + 1
+    out_tensor = model.predict([inp1_tensor, inp2_tensor])
+    # Save model.
+    dataset_gen = DatasetGenerator([inp1_tensor, inp2_tensor])
```

Comment:
Is `DatasetGenerator` defined somewhere? Are you using `dataset_gen` later in the code?

Reply:
Removed